internal/pkg/skuba/deployments/ssh/ssh.go

```diff
@@ -122,6 +122,7 @@ func (t *Target) GetDeployment(nodename string, role *deployments.Role, verboseL
 		port:         t.port,
 		verboseLevel: verboseLevel,
 	}
+	res.OSRelease()
```

Comment:
Why would you include that in GetDeployment? Is there a reason why not doing it immediately in the target call that needs it? Like the upgrade apply/plan?